setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
How is this file used? I don't understand how travis or the project uses it in order to set these rules when checking pep8.